### DISCUSSION TITLE
Using Batch editor to make Box of Mega Pokemon

### DISCUSSION THREAD

**Discussion by Phyore** [kwsch/PKHeX#3777] [Q&A]
```
=Box=31
.Species=$1, 719
.Ability=$1
.Form=1
```

I am using the above to and create a box full of Mega Pokemon. The problem I am encountering is it creates Pokemon that are not Mega (This is because the `Form=1` != Mega but the first form in the list. Is there a way to generate only Mega capable Pokemon?

Alternatively is there a way to randomize between set species? e.g. `$(3,6,9)` to only select Venasaur, Blastoise or Charizard.

Many thanks

**Comment by kwsch** [comment]
1. The batch editor edits existing data.
2. Not all mega forms are form 1. Charizard and Mewtwo two mega forms.

If you want a custom routine to create new data or randomize from a selection, then it is better to just write your own C# code to do what you want. Just fudging data with the batch editor really won't get you anything clean.
